socialNetwork/src/UserTimelineService/UserTimelineService.cpp

```diff
@@ -112,9 +118,24 @@ int main(int argc, char *argv[]) {
                            server_socket,
                            std::make_shared<TFramedTransportFactory>(),
                            std::make_shared<TBinaryProtocolFactory>());
-    LOG(info) << "Starting the user-timeline-service server with Redis Cluster support...";
+    LOG(info) << "Starting the home-timeline-service server with Redis Cluster support...";
```

Comment:
it's not home-timeline-service, fix message, it should be `user-timeline-service`